democracy/migrations/0032_add_language_code_to_comment.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2016-12-16 15:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('democracy', '0031_remove_untranslated_fields'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='sectionimage',
+            options={'ordering': ('ordering', 'translations__title'), 'verbose_name': 'section image', 'verbose_name_plural': 'section images'},
+        ),
+        migrations.AddField(
+            model_name='sectioncomment',
+            name='language_code',
+            field=models.CharField(blank=True, max_length=15, verbose_name='language code'),
+        ),
```

Comment:
Let's add language detection of existing comments (mostly in Finnish, Swedish and English) here, so just run `comment._detect_lang()` for all the data?